mayastor/src/bdev/nexus/nexus_child.rs

```diff
@@ -149,8 +135,8 @@ pub struct NexusChild {
     #[serde(skip_serializing)]
     pub(crate) desc: Option<Arc<Descriptor>>,
     /// current state of the child
-    pub(crate) state: ChildState,
-    pub(crate) status_reasons: StatusReasons,
+    #[serde(skip_serializing)]
+    state: RefCell<ChildState>,
```

Comment:
Why do we need RefCell here?

Reply:
I've removed it - it was for a previous version of the changes but I don't think it is necessary now.

Reply:
:+1: